website/docs/docusaurus-core.mdx

```diff
@@ -334,6 +334,12 @@ You can even omit the children prop and specify a translation string in your `co
 
 :::
 
+:::info
+
+Your custom uses of `<Translate />` are also capable of handling [string pluralization](./i18n/i18n-tutorial.mdx#pluralization)
```

Comment:
```suggestion
The `<Translate>` component supports [interpolation](#interpolate) and [pluralization](./i18n/i18n-tutorial.mdx#pluralization).
```

Reply:
Hmmm, do we really support pluralization when using `<Translate>`? Afaik we don't.

Also we probably need to apply `selectMessage()` before translating the content 😅 otherwise we translate string parts that we don't even use.

Reply:
Thank you